api/src/pcapi/connectors/dms/serializer.py

```diff
@@ -130,7 +130,7 @@ def parse_beneficiary_information_graphql(
         last_name=last_name,
         phone=phone,
         postal_code=postal_code,
-        procedure_id=procedure_id,
+        procedure_id=application_detail.procedure.number,
```

Comment:
procedure_number=...

Dans DMSContent, tu peux utiliser le même hack pour la compatibilité avec les vieilles données que `application_number: int = pydantic.Field(..., alias="application_id")  # keep alias for old data`

Reply:
Yes pourquoi pas
Je sais pas si on veut faire ça dans cette PR ou dans le ticket dédié à la refacto de application_id